src/HttpClientFactory/Http/src/Logging/LoggingHttpMessageHandlerBuilderFilter.cs

```diff
@@ -41,12 +50,15 @@ public Action<HttpMessageHandlerBuilder> Configure(Action<HttpMessageHandlerBuil
                 var outerLogger = _loggerFactory.CreateLogger($"System.Net.Http.HttpClient.{loggerName}.LogicalHandler");
                 var innerLogger = _loggerFactory.CreateLogger($"System.Net.Http.HttpClient.{loggerName}.ClientHandler");
 
+                var options = _optionsMonitor.Get(builder.Name);
+                var logSensitiveHeaders = new ReadOnlyCollection<string>(_optionsMonitor.Get(builder.Name).LogSensitiveHeaders);
```

Comment:
Duplicate call to `_optionsMonitor.Get(...)` here.

Reply:
😊